moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -106,6 +112,7 @@ const SearchBar = ({
   filtersConfig,
 }) => {
   const classes = useStyles();
+  let [, setSearchParams] = useSearchParams();
```

Comment:
im not familiar with this pattern of `let [, something] =`, what does that do?

Reply:
just caught this comment. It is a way to skip an item when destructuring an array. [This post](https://www.samanthaming.com/tidbits/13-skip-values-in-destructuring/) explains it way better than I could! If I assigned the first item (which is a getter for `searchParams`), I would get a compile warning about an unused variable.

Reply:
TIL! I wouldn't have thought in a hundred years `,,,,` could be a valid bit of javascript, and there it is. Thank you so much for sharing that link, it's so helpful.